source/features/show-associated-branch-prs.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import cache from 'webext-storage-cache';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {isForkedRepo} from '../libs/page-detect';
+import * as pageDetect from '../libs/page-detect';
+import pullRequestIcon from 'octicon/git-pull-request.svg';
+import {getRepoURL, getRepoGQL} from '../libs/utils';
+
+const getOpenPullRequests = cache.function(async (): Promise<AnyObject> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			refs(refPrefix: "refs/heads/", last: 100) {
+				nodes {
+					name
+					associatedPullRequests(last: 1, orderBy: {field: CREATED_AT, direction: DESC}) {
+						nodes {
+							number
+							state
+							url
+						}
+					}
+				}
+			}
+		}
+	`);
+	const pullRequests = {};
+	for (const branches of repository.refs.nodes) {
+		if (branches.associatedPullRequests.nodes.length > 0) {
+			pullRequests[branches.name] = branches.associatedPullRequests.nodes[0];
+		}
+	}
+
+	return pullRequests;
+}, {
+	maxAge: 1,
+	cacheKey: () => 'associatedBranchPullRequests:' + getRepoURL()
+});
+
+function capitalizeFirstLetter(string: string): string {
```

Comment:
@fregante whats a better way to do this?

Reply:
That's pretty much it.

However you can just use CSS since it's just one word: `text-transform: capitalize;`

Reply:
Didnt work since the text is not inside the element.

Reply:
I just tried. It works. Apply it to `.State`

Reply:
I saw you're also using it in the `title` attribute, so this can stay.